node/sc/subbridge.go

```diff
@@ -514,15 +519,45 @@ func (pm *SubBridge) handle(p BridgePeer) error {
 
 			if pm.peers.Len() == 1 {
 				pm.handler.setMainChainAccountNonceSynced(false)
```

Comment:
This Len() is not atomic. Let's move this routine into resetBridgeLoop() also.

Reply:
Fixed.